docs/handbook/image-file-formats.rst

```diff
@@ -551,6 +551,119 @@ The :py:meth:`~PIL.Image.Image.save` method supports the following options:
     library before building the Python Imaging Library. See the `installation
     documentation <../installation.html>`_ for details.
 
+APNG sequences
+~~~~~~~~~~~~~~~~~
+
+The PNG loader includes limited support for reading and writing APNG files.
+When an APNG file is loaded, :py:meth:`~PIL.ImageFile.ImageFile.get_format_mimetype`
+will return ``"image/apng"``, and the :py:attr:`~PIL.Image.Image.is_animated` property
+will be ``True`` (even for single frame APNG files).
```

Comment:
I'm not currently in favour of `is_animated` returning true for only one frames. You've said that this is to 'differentiate single frame APNG files from standard PNG files'. However, you can already tell the two apart through the mimetype. Thoughts?

Reply:
I wrote it that way because in my head I was associating `is_animated` with "is the `seek()` method available", and for single frame APNG files `seek(1)` is valid if there is a separate default image in addition to the first (and only) animation frame.

But I'm fine with changing it to make the behavior more consistent with other image formats.

Reply:
Just to double check this before I make any changes, my understanding is that the desired behavior is for `is_animated` to be true if `Image.n_frames` is greater than 1.

Meaning:

* If the APNG contains one frame and no default image, `is_animated` should be false (since `seek(0)` would return the first frame, and `seek(1)` is invalid)
* If the APNG contains one frame as well as a default image, `is_animated` should be true (since `seek(0)` would return the default image, and `seek(1)` is valid and returns the first frame)
* If the APNG contains more than one frame, `is_animated` should be true (the presence of a default image is irrelevant for this case)

Reply:
Resolved in ff7aea0, `is_animated` is now true when `n_frames` is greater than 1, and false otherwise (`n_frames` still depends on both animation frame count as well as presence of a separate default image)